learning-ojs/en/introduction.md

```diff
@@ -31,7 +31,7 @@ OJS includes the following features:
 8. Complete context-sensitive online Help support  
 9. Multilingual support
 
-## What's New in OJS 3.1
+## What's New in OJS 3
 
 OJS 3.1 builds on the changes introduced with OJS 3, and is significantly different than its predecessor, OJS 2. It includes enhancements and new features developed from community feedback, extensive usability testing, and new software design capabilities.
```

Comment:
This sentence is phrased like the section is specifically about 3.1. Maybe that needs to be changed as well?